integrations/snyk/snyk/client.py

```diff
@@ -66,19 +65,8 @@ async def _send_api_request(
             return response.json()
 
         except httpx.HTTPStatusError as e:
-            response_json = e.response.json()
-            if not raise_for_snyk_error and any(
-                error.get("code") == "SNYK-9999"
-                for error in response_json.get("errors", [])
-            ):
-                logger.error(
-                    f"Encountered Synk internal error while sending request: method: {method}, "
-                    f"url: {url}, query_params: {query_params}, version: {version}, json: {json_data}"
-                )
-                return {}
-
             logger.error(
-                f"HTTP error with status code: {e.response.status_code} and response: {response_json}"
+                f"HTTP error with status code: {e.response.status_code} and response: {e.response.text}"
```

Comment:
I would still like to have more context for when experiencing errors, like what url etc..